src/components/Section.js

```diff
@@ -1,15 +1,22 @@
 export default class Section {
-  constructor({ items, renderer }, containerSelector) {
-    this._rendereritems = items;
+  constructor({ items, renderer }, cssSelector) {
+    this._items = items;
     this._renderer = renderer;
-    this._class = document.querySelector(containerSelector);
+    this._container = cssSelector;
```

Comment:
The `selector` word means a **string** that is used for searching elements in the DOM

```js
    this._container = document.querySelector(cssSelector);;

```